### ISSUE TITLE
add conversion script from 0.8 to 1.0

### ISSUE THREAD

**Issue by kain88-de** [datreant/MDSynthesis#74]
Have a script that I can run on a folder with mdsynthesis `<=0.8` folders and convert them to the new datreant format automatically.

**Comment by kaceyaurum** [comment]
I could use this as well! @dotsdl

**Comment by kaceyaurum** [comment]
The following worked for me.
```Python
import mdsynthesis as mds
import json
from os import path
from glob import glob

def convert(folder):
    json_file = glob(path.join(folder, 'Sim*'))[0]
    with open(json_file) as fh:
        old = json.load(fh)
    mds.Sim(folder, categories=old['categories'], tags=old['tags'])
```

**Comment by kaceyaurum** [comment]
Added pull request to fix #74; could someone have a look at it, please?
